just-install.json

```diff
@@ -20,6 +20,22 @@
       },
       "version": "19.00"
     },
+    "adoptopenjdk-13-jdk": {
+      "installer": {
+        "kind": "msi",
+        "x86": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.1%2B9/OpenJDK13U-jdk_x86-32_windows_hotspot_13.0.1_9.msi",
+        "x86_64": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.1%2B9/OpenJDK13U-jdk_x64_windows_hotspot_13.0.1_9.msi"
+      },
+      "version": "jdk-13.0.1+9"
+    },
+    "adoptopenjdk-13-jre": {
+      "installer": {
+        "kind": "msi",
+        "x86": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.1%2B9/OpenJDK13U-jre_x86-32_windows_hotspot_13.0.1_9.msi",
+        "x86_64": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.1%2B9/OpenJDK13U-jre_x64_windows_hotspot_13.0.1_9.msi"
+      },
+      "version": "jdk-13.0.1+9"
```

Comment:
Is this version being `jdk-*` for both the JDK and JRE intentional? If so, I think it would be better to remove the `jdk-` prefix?

Reply:
OK. Will change.

Reply:
Changed